lib/src/test/kotlin/org/didcommx/didcomm/diddoc/DIDDocEncodingTest.kt

```diff
@@ -0,0 +1,49 @@
+/*-
+ * #%L
+ * Nessus DIDComm :: Core
```

Comment:
Let's remove references to [https://github.com/tdiesler/nessus-didcomm](nessus-didcomm)

Reply:
Thanks ... header removed